### ISSUE TITLE
Application crashes on Linux when loading specific images with SWT version >4.9

### ISSUE THREAD

**Issue by kahlep** [Transkribus/TranskribusSwtGui#322]
Happens with `SWT_GTK3=0` as well as `SWT_GTK3=1`. Tested with current Oracle JDK 8 and OpenJDK 14

```
13:41:59.850 DEBUG [pool-1-thread-1] eu.transkribus.swt_gui.canvas.CanvasImage - Trying to dispose imgOrig: null
13:41:59.850 DEBUG [pool-1-thread-1] eu.transkribus.swt.util.SWTUtil - Disposing SWT image: null
[xcb] Unknown sequence number while processing reply
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
java: xcb_io.c:641: _XReply: Zusicherung »!xcb_xlib_threads_sequence_lost« nicht erfüllt.
```

Maybe related:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=495896
https://bugs.launchpad.net/ubuntu/+source/gtk+2.0/+bug/1808710
pointing to this bug in libX11: https://gitlab.freedesktop.org/xorg/lib/libx11/-/merge_requests/34

**Comment by kahlep** [comment]
Got another error message using OpenJDK 14 and SWT 4.15 (the above was with SWT 4.17):
```
[xcb] Unknown request in queue while dequeuing
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
java: xcb_io.c:163: dequeue_pending_request: Zusicherung »!xcb_xlib_unknown_req_in_deq« nicht erfüllt.
```

**Comment by jkloe** [comment]
Problem solved using SWT 4.14
